components/InstallModal.js

```diff
@@ -26,7 +29,13 @@ export default class extends React.Component {
             Use the <code>hyper</code> command, bundled with your Hyper app, to
             install {this.props.name} by entering the following into Hyper:
           </p>
-          <pre>hyper i {this.props.name}</pre>
+          <pre
+            ref={pre => {
+              this.installCode = pre
+            }}
```

Comment:
It will recreate a function at each render.
Maybe better to move this to an `onInstallCodeRef()` method, binded to `this` in constructor.
This is obviously not a performance bottleneck 😀 
But maybe cleaner.